Gemfile

```diff
@@ -56,6 +56,7 @@ group :development, :test do
   gem 'capistrano-passenger'
   gem 'capistrano-rails', '~> 1.6', require: false
   gem 'factory_bot'
+  gem 'rails-controller-testing'
```

Comment:
Is this new gem needed?  I didn't see any controller tests as part of this PR.

Reply:
This gem is needed for the `render_template(:new)` tests